modulus/models/layers/fused_silu.py

```diff
@@ -15,7 +15,11 @@
 import functools
 import torch
 from torch.autograd import Function
-from nvfuser._C import Fusion, FusionDefinition, DataType
+
+try:
+    from nvfuser._C import Fusion, FusionDefinition, DataType
+except:
+    from torch._C._nvfuser import Fusion, FusionDefinition, DataType
```

Comment:
This is to accommodate earlier versions of PyTorch? Would also be good to specify an `ImportError` exception here

Reply:
agreed